Laravel.gitignore

```diff
@@ -21,3 +21,15 @@ Homestead.yaml
 Homestead.json
 /.vagrant
 .phpunit.result.cache
+
+/public/build
+/storage/pail
+.env.backup
+.env.production
+.phpactor.json
+auth.json
+/.fleet
+/.idea
+/.nova
+/.vscode
+/.zed
```

Comment:
Hey @moser-jose 
Could you remove the non framework specific lines from this? `.vscode`, `.idea` and such are not specific to Laravel and are generally kept in their own file elsewhere in the repo

Reply:
done removed, thanks.